### PR TITLE
translation for cordova

### PR DESCRIPTION
**Pull request by lisa0314**
@jondong ,@shaoboyang, @chuan9,please help to review.

### CODE REVIEW

File: public/documentation/cordova_zh.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# 支持Cordova 4.0的Crosswalk
+## 概述
+随着[Cordova Android 4.0.0](http://cordova.apache.org/announcements/2015/04/15/cordova-android-4.0.0.html)引入了对插入式webview的支持，现在你可以方便地在你的Cordova应用上使用Crosswalk的webview。通过使用Crosswalk的webview插件，开发者可以享用远程调试的功能，前沿的HTML5特性，例如WebGL, WebAudio和WebRTC，以及在包括Android 4.0 Ice Cream Sandwich(ICS)在内的Android设备上性能的显著提升。
+## 先决条件
+请参见[Android平台指导](https://cordova.apache.org/docs/en/4.0.0/guide_platforms_android_index.md.html#Android%20Platform%20Guide)建立起针对Cordova应用开发的Android SDK环境。
+
+确保你的系统是否已经安装了最新版本的[node.js](https://nodejs.org/)。
+## 工作流程
+
+1.  安装Cordova命令行接口（CLI）
```
Comment (jondong) [inline public/documentation/cordova_zh.md]: 命令行工具

File: public/documentation/cordova_zh.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        /path/to/hello/platforms/android/build/outputs/apk/android-x86-debug.apk
+        /path/to/hello/platforms/android/build/outputs/apk/android-armv7-debug.apk
+
+    至此，Crosswalk WebView库将会被嵌入到你的应用中。这大概会使APK的大小增加18MB。
+
+7.  在模拟器上启动它
+
+        $ cordova emulate android
+
+    使用Crosswalk WebView启动一个Cordova应用的做法和直接使用Cordova应用差不多。上述命令行将会启动模拟器，安装应用的APK，然后启动你的应用。例如，在启动上述的Cordova样例后，你将在你的屏幕上看到下列的结果：
```
Comment (jondong) [inline public/documentation/cordova_zh.md]: 样例应用后

File: public/documentation/cordova_zh.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    <img src="/assets/cordova-in-emulator.jpg" />
+
+    关于详细的命令行和选项，请参见[命令行接口](https://cordova.apache.org/docs/en/4.0.0/guide_cli_index.md.html#The%20Command-Line%20Interface)。
+
+
+## 通过Crosswalk远程调控
+
+Crosswalk的WebView允许远程调试Cordova应用，即使是较旧的Android设备(4.0+)。
```
Comment (jondong) [inline public/documentation/cordova_zh.md]: 即使是在较旧的Android设备上（4.0+）也可以。

File: public/documentation/cordova_zh.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+关于使用手册请参见[使用Chrome远程调试Android应用](https://developer.chrome.com/devtools/docs/remote-debugging)。
+
+
+## (可选) 仅针对Android平台的工作流
+
+这个工作流主要针对那些对前沿开发技术感兴趣的开发者。如果你想要一个稳定版本，可以参考Cordova CLI工作流。
+
+需要[git](http://www.google.com/url?q=http%3A%2F%2Fgit-scm.com%2F&sa=D&sntz=1&usg=AFQjCNFOqwvh2KbuCJQUVsR5fW38FrTRTw)。
+
+1.  复制Crodova-android的资源库
```
Comment (jondong) [inline public/documentation/cordova_zh.md]: Pull cordova-android代码库。

File: public/documentation/cordova_zh.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    这样将会自动去Crosswalk项目发布网站(https://download.01.org/crosswalk/releases/crosswalk/android/)获取Crosswalk WebView库，并且可以为X86和ARM架构编译。例如，它会产生：
+
+        /path/to/hello/build/outputs/apk/hello-x86-debug.apk
+        /path/to/hello/build/outputs/apk/hello-armv7-debug.apk
+
+8.  在模拟器上启动它
+
+        $ ./cordova/run --emulator --nobuild
+
+    这个启动了模拟器，安装了应用的APK并启动了你的应用。
```
Comment (jondong) [inline public/documentation/cordova_zh.md]: 这条命令启动了...

**Comment by jondong** [comment]
@lisa0314 Thanks for the PR, some translation nits.

**Comment by lisa0314** [comment]
@jondong  Thanks for your review. All suggestions done.

**Comment by jondong** [comment]
LGTM. merging

**Pull request merged**